python/nnvm/frontend/darknet.py

```diff
@@ -4,6 +4,7 @@
 
 from __future__ import absolute_import as _abs
 from enum import IntEnum
+from cffi import FFI
```

Comment:
do not include cffi dependency in this file, otherwise this will make the nnvm compiler depend on cffi

Reply:
@tqchen cffi dependancy is removed from the frontend interfaces. please have a look again.